aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/credentails/CognitoCredentialsProvider.kt

```diff
@@ -74,6 +76,15 @@ internal class CognitoCredentialsProvider : AuthCredentialsProvider {
     }
 }
 
+private fun AWSCredentials.toCredentials(): Credentials {
```

Comment:
Would probably be best to place this inside AWSCredentials. Since we own the class, we don't need an extension method.

Reply:
Will do in follow up PR when we move this class to core.